config/recipes/logstash/logstash-eck.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: elasticsearch
+  labels:
+    app.kubernetes.io/name: eck-logstash
+    app.kubernetes.io/component: elasticsearch
```

Comment:
I think I would remove those labels to propose a minimal recipe and stay focus on what we want to show here. I see that they were in the old recipe `logstash.yml` but we do not set them in all the others recipes.